code/datums/diseases/appendicitis.dm

```diff
@@ -7,9 +7,8 @@
 	cure_text = "Surgery"
 	agent = "Shitty Appendix"
 	viable_mobtypes = list(/mob/living/carbon/human)
-	permeability_mod = 1
 	desc = "If left untreated the subject will become very weak, and may vomit often."
-	severity = "Dangerous!"
```

Comment:
this better not have been why appendicitis never rolls

Reply:
(Un?)fortunately Appendicitis is its own event, so it should roll fine

Reply:
Appendicitis is a moment, it also can only roll once per round so it's very uncommon.